scripts/zkai_d64_recursive_pcd_aggregation_feasibility_gate.py

```
@@ -0,0 +1,732 @@
+    return payload
+
+
+def build_payload() -> dict[str, Any]:
+    return _build_payload_from_canonical_source(load_checked_block_receipt())
+
+
+def _validate_source_descriptor(payload: dict[str, Any]) -> dict[str, Any]:
+    descriptor = require_object(payload.get("source_block_receipt_evidence"), "source block receipt evidence")
+    path_value = descriptor.get("path")
+    if not isinstance(path_value, str):
+        raise D64RecursivePCDFeasibilityError("source block receipt evidence path must be a string")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)